Classification/Core/src/main/java/org/tribuo/classification/ensemble/AdaBoostTrainer.java

```diff
@@ -136,12 +137,24 @@ public String toString() {
      * @return A {@link WeightedEnsembleModel}.
      */
     @Override
-    public Model<Label> train(Dataset<Label> examples, Map<String, Provenance> runProvenance) {
+    public WeightedEnsembleModel<Label> train(Dataset<Label> examples) {
+        return train(examples, Collections.emptyMap(), INCREMENT_INVOCATION_COUNT);
+    }
+
+    /**
+     * If the trainer implements {@link WeightedExamples} then do boosting by weighting,
+     * otherwise do boosting by sampling.
+     * @param examples the data set containing the examples.
+     * @param runProvenance Provenance information specific to this run.
+     * @return A {@link WeightedEnsembleModel}.
+     */
+    @Override
+    public WeightedEnsembleModel<Label> train(Dataset<Label> examples, Map<String, Provenance> runProvenance) {
         return(train(examples, runProvenance, INCREMENT_INVOCATION_COUNT));
```

Comment:
```suggestion
        return train(examples, runProvenance, INCREMENT_INVOCATION_COUNT);
```

Reply:
Thanks, I dunno why IntelliJ started doing that when I was adding things.